pom.xml

```diff
@@ -258,6 +258,7 @@
         <module>legend-engine-xt-nonrelationalStore-mongodb-grammar</module>
         <module>legend-engine-xt-nonrelationalStore-mongodb-protocol</module>
         <module>legend-engine-xt-nonrelationalStore-mongodb-grammar-integration</module>
+        <module>legend-engine-entitlement</module>
```

Comment:
as discussed, we would like to split this module into 2 part: core logic and API logic, since the core logic can be depended on by other stores, we don't want to mingle in the API depeendencies (like dropwizard for example)

Reply:
Also, we don't we expose the API in legend-engine-server?